tests/h/search/query_test.py

```diff
@@ -938,3 +975,10 @@ def es_dsl_search(pyramid_request):
         using=pyramid_request.es.conn,
         index=pyramid_request.es.index,
     )
+
+
+@pytest.fixture
+def AnnotationSearchIndexPresenter(patch):
+    class_ = patch('h.search.index.presenters.AnnotationSearchIndexPresenter')
+    class_.return_value.asdict.return_value = {'test': 'val'}
+    return class_
```

Comment:
Usually in patch fixtures we use the name of the module or class being patched as the name of the local variable. `AnnotationSearchIndexPresenter` in this case. This avoids the need to make up names like `class_` or `module` or `svc` or `obj` etc:

```python
@pytest.fixture
def AnnotationSearchIndexPresenter(patch):
    AnnotationSearchIndexPresenter = patch("h.search.index.presenters.AnnotationSearchIndexPresenter")
    AnnotationSearchIndexPresenter.return_value.asdict.return_value = {"test": "val"}
    return AnnotationSearchIndexPresenter
```